options/locale/locale_en-US.ini

```diff
@@ -1994,6 +1994,8 @@ settings.event_package = Package
 settings.event_package_desc = Package created or deleted in a repository.
 settings.branch_filter = Branch filter
 settings.branch_filter_desc = Branch whitelist for push, branch creation and branch deletion events, specified as glob pattern. If empty or <code>*</code>, events for all branches are reported. See <a href="https://pkg.go.dev/github.com/gobwas/glob#Compile">github.com/gobwas/glob</a> documentation for syntax. Examples: <code>master</code>, <code>{master,release*}</code>.
+settings.authorization_header = Authorization Header
+settings.authorization_header_desc = "If it's not empty, it will be included as an authorization header to the webhook request. Examples: %s."
```

Comment:
@Gusted might have an opinion here as well, since he [previously commented](https://github.com/go-gitea/gitea/pull/20926#discussion_r956584965)

 (I prefer your suggestion)